tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 isolated_build = True
-envlist = fmt, lint, mypy, checks, py{3.8,3.9,3.10,3.11}-django{32,41,42,main}
+envlist = fmt, lint, mypy, checks, py{3.10,3.11}-django{32,41,42,main}
```

I've had another look at the releases - as I don't think the grid in poetry-template is correct. I also think that supporting point releases between LTS releases is overkill - if it works with 3.2 and 4.2 I think we should assume it will work for 4.0, 4.1 - at least until proven otherwise?

```suggestion
envlist =
    fmt, lint, mypy, checks,
    py{3.10}-django{32,42,50,main}
    py{3.11}-django{42,50,main}
    py{3.12}-django{50,main}
```

---

@hugorodgerbrown I've added these changes both to `tox.ini` and to the github actions workflow file.